source/config-file/specify-connection-options.txt

```diff
@@ -0,0 +1,81 @@
+.. _compass-specify-connection-options:
```

Comment:
Just a note and it doesn’t affect me personally anyway, but since this page focuses on read preferences and tags (as the title says), maybe the file name/id here should also be adjusted to match the title?